modules/ROOT/pages/netsuite/netsuite-examples.adoc

```diff
@@ -0,0 +1,567 @@
+= NetSuite Connector Examples - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+NetSuite Connector Version 9.2
+
+The following examples show how to perform searches in Anypoint Connector for NetSuite (NetSuite Connector):
+
+* <<account-basic-search,Account Basic Search Filtering by an Integer Field>>
+* <<employee-basic-search,Employee Basic Search Filtering by the Boolean and String Fields>>
+* <<customer-advanced-search,Customer Advanced Search Filtering by the Date and Array Fields>>
+
+[[account-basic-search]]
+== Account Basic Search Filtering by an Integer Field
+
+This example shows how to use the NetSuite Search operation to obtain the Account Basic items with internal IDs that are less than the value of the `query` parameter.
+
+////
+Is "items" the correct terminology in the above paragraph or should we say "records"?
```

Comment:
"Records" is better. Also, what you get are Account (not Account Basic) records. What is basic is the type of query you run against NetSuite.
Yes, it's a weird API...